widget/toolbar.go

```diff
@@ -68,14 +68,102 @@ func NewToolbarSeparator() ToolbarItem {
 // Toolbar widget creates a horizontal list of tool buttons
 type Toolbar struct {
 	BaseWidget
-	Items []ToolbarItem
+	Items    []ToolbarItem
+	focused  bool
```

Comment:
I don't understand why a toolbarButton should not be focused but the toolbar can be?
This code generally seems strange - why would a bar track which of it's non-focusable children is focused?

Reply:
In the various apps I have tried it seems that toolbars are not focusable.
If that's not right then we can come back to it.

Reply:
You are right, but it makes for a lot of Tab presses to navigate through the tab bar. If a toolbar has 10 toolbar buttons, you must press Tab 10 times to go to the first entry in the form. I like it much better with the chosen approach, where the Tab continues to the next object and arrow keys are used to select the actual toolbar button. It is very similar to the tabbed container.

Reply:
Looking at more applications I see that toolbars are not focusable at all.
It's like they only focus "input controls" and not every element of the interface.
I don't know if that is right either - but if we are to focus toolbar not items then it should probably appeare focused rather than an item in it?

Reply:
Usualy the toolbar buttons are associated with keyboard shortcuts, and then there is no need to focus them. To do this we need an obligatory shortcut for each toolbar button. I leave this for future updates.

Reply:
That is a good idea, apart from the mandatory part. Some work for Shortcuts is in place, hopwfully we can make use of that.

Reply:
IMHO, a toolbar (or containing buttons) **never** should be focusable. Toolbars are for easy clicking with the mouse. All commands available in a toolbar should also be available as a menu item in the menu bar - hence keyboard-only users already use the menu or its accelerator.

Reply:
I agree. But it depends on having keyboard shortcuts. And the menu must be focusable.

Reply:
It would be great to get this split into smaller pieces so we can see some of this functionality land - the conflict list is probably too long to get this PR landed in one go.